src/parse.c

```diff
@@ -1624,6 +1631,22 @@ const mapping_entry_handler tunnel_def_handlers[] = {
      */
     {"key", YAML_NO_NODE, handle_tunnel_key_mapping},
     {"keys", YAML_NO_NODE, handle_tunnel_key_mapping},
+
+    /* l2tp; reuses tunnel.local_ip and tunnel.remote_ip*/
+    {"local_tunnel_id", YAML_SCALAR_NODE, handle_netdef_guint, NULL, netdef_offset(l2tp.local_tunnel_id)},
```

Comment:
Names for keys are typically split with hyphen rather than underscore. Could you please adjust that so things are at least consistent in that regard?

Reply:
Oh, how could I miss that.
Done.